README.md

```diff
@@ -177,11 +178,27 @@ class User < ActiveRecord::Base
 end
 ```
 
-There's no technical reason why you couldn't detect enum columns at startup time and automatically do this wireup, but I feel that the benefit of self-documenting outweighs the convenience.
+Alternatively values can be inferred from the enum values using the `inferred_enum` helper, which can be helpful for readability especially if the type contains a large number of values.
+
+For example:
+```ruby
+class ContactInfo < ActiveRecord::Base
+  inferred_enum :contact_method, :contact_method_type
```

Comment:
The inferred_enum method seems to be defined on the connection and not on the ActiveRecord model.